src/h5web/visualizations/heatmap/LogScaleToggler.tsx

```diff
@@ -1,23 +1,25 @@
 import React from 'react';
-import { FiEyeOff, FiEye } from 'react-icons/fi';
-import styles from './HeatmapVis.module.css';
-import { useHeatmapState, useHeatmapActions } from './store';
+import { FiToggleLeft, FiToggleRight } from 'react-icons/fi';
+import styles from './HeatmapToolbar.module.css';
+import { useHeatmapStore } from './store';
 
 function LogScaleToggler(): JSX.Element {
-  const { hasLogScale } = useHeatmapState();
-  const { toggleLogScale } = useHeatmapActions();
+  const [hasLogScale, toggleLogScale] = useHeatmapStore(state => [
+    state.hasLogScale,
+    state.toggleLogScale,
+  ]);
```

Comment:
if you use this syntax, you need to pass `shallow` so the return value is compared shallowly https://github.com/react-spring/zustand#selecting-multiple-state-slices

Reply:
:fishing_pole_and_fish: